docs/Configuration.md

```diff
@@ -363,6 +363,64 @@ Any public field or setter method in this class can be given a default value usi
 }
 ```
 
+## Routing modes
+
+The routing request parameter `mode` determines which transport modalities should be considered when calculating the list
+of routes.
+
+Some modes (mostly bicycle and car) also have optional qualifiers `RENT` and `PARK` to specify if vehicles are to be parked at a station or rented. In theory
+this can also apply to other modes but makes sense only in select cases which are listed below.
+
+Whether a transport mode is available highly depends on the input feeds (GTFS, OSM, bike sharing feeds) and the graph building options supplied to OTP.
+
+The complete list of modes are:
+
+- `WALK`: Walking some or all of the route.
+
+    _Prerequisite_: The maximum walking distance configured has not been exceeded.
+
+- `TRANSIT`: General catch-all for all public transport modes.
+
+- `BICYCLE`: Cycling for the entirety of the route or taking a bicycle onto the public transport and cycling from the arrival station to the destination.
+
+- `BICYCLE_RENT`: Taking a rented, shared-mobility bike for part or the entirety of the route.  
+
+    _Prerequisite:_ Vehicle positions need to be added to OTP either as static stations or dynamic data feeds. 
+
+    For dynamic bike positions configure an input feed. See [Configuring real-time updaters](Configuration.md#configuring-real-time-updaters).
+
+    For static stations check the graph building documentation for the property `staticBikeRental`.
+
+- `BICYCLE_PARK`: Leaving the bicycle at the departure station and walking from the arrival station to the destination. 
+
+    _Prerequisite:_ Bicycle parking stations near the station and visible to OTP by enabling the property `staticBikeParkAndRide` during graph build.
+
+- `CAR`: Driving your own car the entirety of the route.
+
+- `CAR_PARK`: Driving a car to the park-and-ride facilities near a station and taking public transport.
+
+    _Prerequisite:_ Park-and-ride areas near the station and visible to OTP by enabling the property `staticParkAndRide` during graph build.
```

Comment:
The `staticParkAndRide` is set to true by default. It might also be worth noting that there need to be Park and Rides defined in the OpenStreetMap data that is being used.

Reply:
Also, it should be noted that for Park & Ride searches at least one transit mode should also be included, otherwise this will behave as if only the CAR mode were sent over.